src/employee_management/Department.java

```diff
@@ -1,8 +1,12 @@
 package employee_management;
+import java.util.List;
 
 /**
  * TODO : a department is a collection of employees. It has a name and a set of employees in it.
  */
+
+
+
 public class Department {
```

Comment:
How do you prevent duplicates of employee in the departement ?

Reply:
I will now use maybe Set instead of lists..